app/assets/javascripts/replies.js

```diff
@@ -0,0 +1,30 @@
+jQuery(function() {
+
+  var current_selection = "";
+
+  jQuery(document).on('mouseup', 'body', function() {
+    current_selection = window.getSelection().toString();
+  })
+
+  jQuery('.split-off-ticket').bind('click', function(event) {
+    if (current_selection != "") {
+      var url = $(this).attr('href') + ".json";
```

Comment:
Also, this file seems to contain single and double quote strings. Could you modify them to be single quotes everywhere?